code/game/atoms_movable.dm

```diff
@@ -80,10 +80,10 @@
 		last_move = 0
 		return
 
+	last_move = direct

```

Comment:
Moving it here is alright; double-checked, just in case; you go flying out diagonally when you throw something, in space.